src/support_sphere_py/src/support_sphere/scripts/update_db_sample_data.py

```diff
@@ -46,7 +51,7 @@ def populate_resources(cv_only=False, resource_type_uids: dict[str, uuid.UUID]|N
         if not isinstance(resource_type_uids, dict):
             raise ValueError("resource_type_uids must be provided if cv_only is False")
 
-    file_path = Path("./support_sphere_py/tests/resources/data/resources.csv")
+    file_path = Path(__file__).parent / 'resources' / 'data' / 'resources.csv'
```

Comment:
Looks like you can probably just create a variable that points to the data directory.

Reply:
Done!